components/RichText.tsx

```diff
@@ -0,0 +1,114 @@
+import { documentToReactComponents } from "@contentful/rich-text-react-renderer";
+import { BLOCKS, MARKS, INLINES } from "@contentful/rich-text-types";
+import React from "react";
+import styles from "styles/RichText.module.scss";
+
+type Theme = "LIGHT" | "DARK" | "INDIGO";
```

Comment:
Light for the job post page, dark for the homepage, indigo for the homepage. This should be worth writing documentation about?